### PR TITLE
Upgrade to latest typeorm/pg to support GENERATED ALWAYS AS IDENTITY in migrations

### PR DESCRIPTION
**Pull request by drtyh2o**

### CODE REVIEW

**Review by avatarneil — COMMENTED** [review]
Can these changes be applied in a forward-facing manner, so they can update existing dbs? Would be nice if we could avoid modifying existing migrations, and instead add new ones.

**Comment by drtyh2o** [comment]
> Can these changes be applied in a forward-facing manner, so they can update existing dbs? Would be nice if we could avoid modifying existing migrations, and instead add new ones.

These changes are no-ops. Only made so that new migrations don’t look different from historical ones with the explicit NOT NULL. They do not result in a schema change. New migrations are not required.

**Review by avatarneil — APPROVED** [review]

File: .jest/configurePorts.ts [hunk 1]
```diff
@@ -1,53 +1,40 @@
...
+    'JURISDICTION_SERVICE_RPC_PORT',
+    'POLICY_API_HTTP_PORT',
+    'POLICY_AUTHOR_API_HTTP_PORT',
+    'POLICY_SERVICE_REPL_PORT',
+    'POLICY_SERVICE_RPC_PORT',
+    'RPC_PORT',
+    'TRANSACTION_API_HTTP_PORT',
+    'TRANSACTION_SERVICE_REPL_PORT',
+    'TRANSACTION_SERVICE_RPC_PORT'
+  ].map(port => ({ [port]: `${findPort({ start: 4000, end: 6000 })}` }))
```
Comment (PeterSP) [inline .jest/configurePorts.ts:39]: I would prefer a different variable name than `port` here, as it's not the port but the environment variable name which points to it.

**Review by PeterSP — COMMENTED** [review]
I would prefer renaming that one variable, but otherwise this looks great!!

File: .jest/configurePorts.ts (hunk 1 above)
Comment (drtyh2o) [reply .jest/configurePorts.ts]: Fair point! Fixed in [9cde8b0](https://github.com/lacuna-tech/mds-core/pull/903/commits/9cde8b024541a7be6ad7688df0309fc2ca67bd94)

**Pull request merged**